packages/nuget-exe/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "nuget-exe",
-  "version": "4.9.2",
+  "version": "5.8.0",
```

Comment:
Beachball will bump this to 5.8.1 as is. Changing this to `5.8.0-0` should create the right version though.